tests/helpers/helpers.ts

```diff
@@ -165,6 +185,14 @@ export const createLookupOptions = (amount = randNumber({ min: 1, max: 3 })): IL
   return lookupOptions;
 };
 
+export const createFakeDeleteRequest = (): StoreTriggerDeletePayload => {
+  return {
+    modelId: createUuid(),
+    modelName: randWord(),
+    pathToTileset: linksToString(linksPattern),
```

Comment:
There is no need to create the function linksToString and the linkPattern constant.
You can just do:

```suggestion
    pathToTileset: `${randWord()},${randSentence()},${randWord()},http://${randWord()}.test/wms`
```

Reply:
ok